tests/suites/test_suite_rsa.function

```diff
@@ -470,8 +489,9 @@ void mbedtls_rsa_private( data_t * message_str, int mod, int radix_P,
                               message_str->x, output ) == result );
     if( result == 0 )
     {
-
-        TEST_ASSERT( hexcmp( output, result_hex_str->x, ctx2.len, result_hex_str->len ) == 0 );
+        TEST_ASSERT( result_hex_str->len == ctx.len );
```

Comment:
The commit "tests: Replace use of hexcmp with memcmp" changed `ctx2.len` to `ctx.len`. I think this is a mistake: at this point in the test, we're checking `ctx2`, and `ctx` has been freed. The test now passes only because the library code isn't as robust as it should be: https://github.com/ARMmbed/mbedtls/issues/2024